docs/tempo/website/grafana-agent/span-metrics.md

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
should we completely remove this? I'd prefer leaving it and adding a link to server side span metrics as an alternative. If we do completely remove this we need to provide an alias on another page:

something like: https://github.com/grafana/tempo/blob/main/docs/tempo/website/api_docs/pushing-spans-with-http.md?plain=1#L4

Reply:
Added/